src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,835 @@
+// lipsurf-plugins/src/Netflix/Netflix.ts
+
+/**
+ * Limitations and gotchas:
+ * - This plugin relies on Netflix's `window.netflix` publicized object.
+ *   This plugin MAY fails if netflix introduce a change to that `window.netflix` object
+ * - Watch command relies on videos on cache. It only queries the show netflix has loaded from
+ *   their service and does not query it directly from the service.
+ *
+ * TODOs:
+ * - Remove ContextPatcher stuff when context duplication caused by PluginBase.util.addContext is fixed
+ * - Remove homophone `"search": "search"` after homophone priority issue is fixed on the Platform-land
+ *   https://discuss.lipsurf.com/t/homophones-must-be-a-lower-priority-than-other-plugins-command/297/3
```

Comment:
I think the issue is that the Normal context is taking priority because it's appearing first in the context order. If you explicitly enter the context with Normal coming after the Netflix-specific contexts, I think this will be fixed.  Sorry, docs didn't mention this this so I've updated them: https://docs.lipsurf.com/contexts.html#context-order